daemon/cluster/cluster.go

```diff
@@ -1367,7 +1363,7 @@ func (c *Cluster) CreateNetwork(s apitypes.NetworkCreateRequest) (string, error)
 	defer c.RUnlock()
 
 	if !c.isActiveManager() {
-		return "", c.errNoManager()
+		return "", errors.NewRequestNotAcceptableError(c.errNoManager())
```

Comment:
It is implemented, but the state of the cluster disallows. A 5xx series message is probably correct, as it is both temporary and cannot be resolved by the client.